src/types/netcdfjs.d.ts

```diff
@@ -0,0 +1,2 @@
+// TODO: Hacky way to fix the netcdfjs type error - Adding types through https://github.com/cheminfo/netcdfjs/pull/26
+declare module 'netcdfjs'
```

Comment:
@dkoes This is a very hacky way to fix it, but until we wait for the review regarding my PR at `netcdfjs` package this would be a way